l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.py

```diff
@@ -169,6 +192,21 @@ class PartnerVATList(models.TransientModel):
     total_turnover = fields.Float("Total Turnover", compute="_compute_totals")
     total_vat = fields.Float("Total VAT", compute="_compute_totals")
 
+    @api.multi
+    def _compute_declarant_reference(self):
+
+        self.env["ir.sequence"].next_by_code("declarantnum")
+        company = self.env.user.company_id
+        company_vat = company.partner_id.vat
+
+        if not company_vat:
+            raise ValidationError(_("No VAT number associated with the company."))
+
+        company_vat = company_vat.replace(" ", "").upper()
```

Comment:
Although I can't find the definition of `_format_vat_number`, it should probably be used here for consistency's sake.